deploy/operator.yaml

```diff
@@ -44,3 +44,5 @@ spec:
               value: "false"
             - name: DISABLE_TELEMETRY
               value: "false"
+            - name: PERCONA_VS_FALLBACK_URI
+              value: "https://check.percona.com"
```

Comment:
we shouldn't put this here, the env variable is there to override

Reply:
@nmarukovich  It is only for our tests to overwrite  default  URI if needed. So, we should not set default one.

Reply:
agree